src/server/pps/server/worker_rc.go

```diff
@@ -134,6 +135,25 @@ func (a *apiServer) workerPodSpec(options *workerOptions) api.PodSpec {
 		Volumes:          options.volumes,
 		ImagePullSecrets: options.imagePullSecrets,
 	}
+	// Explicitly set resource requests of the containers to zero.
+	// Some cloud providers have their own default resource requests
+	// which is usually not what we want.
+	for _, container := range podSpec.Containers {
+		cpuQuantity, err := resource.ParseQuantity("0")
```

Comment:
What would you think of using MustParse? You're parsing a constant, so it seems equally safe.